app/src/sharedTest/java/org/oppia/app/topic/overview/TopicOverviewFragmentTest.kt

```diff
@@ -25,13 +29,33 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicOverviewFragmentTest {
 
+  @get:Rule
+  var activityTestRule: ActivityTestRule<TopicActivity> = ActivityTestRule(
+    TopicActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
   @Test
-  fun testTopicOverviewFragment_loadFragment_textIsDisplayed() {
+  fun testTopicOverviewFragment_loadFragmentTest1_topicNameIsDisplayed() {
     ActivityScenario.launch(TopicActivity::class.java).use {
-      onView(withId(R.id.dummy_text_view)).check(matches(withText("This is dummy TextView for testing")))
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText("Second Test Topic")))
     }
   }
 
+  // TODO(#135): Update this test case to check on click of See More play tab is shown.
+  @Test
+  fun testTopicOverviewFragment_loadFragmentTest1_seeMoreIsClickable() {
+    ActivityScenario.launch(TopicActivity::class.java).use {
+      onView(withId(R.id.see_more_text_view)).check(matches(isClickable()))
+    }
+  }
+
+  @Test
+  fun testTopicOverviewFragment_loadFragmentTest1_configuration_change_topicNameIsDisplayed() {
```

Comment:
Nit: ``testTopicOverviewFragment_loadFragment_configurationChange_topicNameIsDisplayed``

We just use ``_``s to split between concepts, not words.

Reply:
Got it. By mistake mixed two different concepts in this. Issue resolved.